mhs/spineroutelookup/lookup/sds_connection_factory.py

```diff
@@ -5,12 +5,15 @@
 
 import definitions
 
+_MAX_RETRIES = 3
```

Comment:
I was looking at code here and a common approach is to load values from environment variables but override with default one if env var is not provided
For example
https://github.com/nhsconnect/integration-adaptors/pull/153/files#diff-8a04b7e2f084c2f292ffdc46a46ccc61R88

Reply:
Done. Added some additional logging also.